nca/FWRules/InteractiveConnectivityGraph.py

```diff
@@ -111,7 +113,7 @@ class SvgGraph:
 
         ELEMENTS_DIRECTORY = 'elements'
 
-        def __init__(self, input_svg_file, output_directory):
+        def __init__(self, input_svg_file, output_directory, expl_xml=None):
```

Comment:
same here

Reply:
done

Reply:
see task for nest PR at:
https://github.com/IBM/network-config-analyzer/issues/662